meinberlin/apps/plans/templates/meinberlin_plans/plan_detail.html

```diff
@@ -24,57 +24,70 @@
                     </li>
                 </ul>
             </nav>
+        </div>
+        <div class="l-center-6">
 
             <div class="item-detail">
                 <h1 class="item-detail__title">
                     {{ object.title }}
                 </h1>
-                <div class="item-detail__meta">{{ object.organisation.name }}</div>
 
                 <div class="item-detail__labels">
                     <span class="label label--secondary">{{ object.get_status_display }}</span>
                     {% if object.category %}
                         <span class="label">{{ object.category }}</span>
                     {% endif %}
-                    {% if object.point_label %}
-                        <span class="label"><i class="fa fa-map-marker" aria-hidden="true"></i> {{ object.point_label }}</span>
-                    {% endif %}
                 </div>
 
-                <div class="item-detail__content">
-                    {% if object.description %}
-                        {{ object.description|richtext }}
-                    {% endif %}
+                <div class="container">
+                    <ul class="item-detail__factsheet">
+
+                        <dl>
+                            <dt>{% trans 'Organisation' %}</dt> <dd>{{ object.organisation.name }}</dd>
+                        </dl>
+
+                        <dl>
+                            <dt>{% trans 'Location' %}</dt><dd>{{ object.point_label|default:'no val' }}</dd>
+                        </dl>
+
+                        <dl>
+                            <dt>{% trans 'Plan type' %}</dt><dd>{{ object.category }}</dd>
+                        </dl>
+
+                        <dl>
+                            <dt>{% trans 'Participation' %}</dt>
+                            <dd>
+                                    {{ object.get_participation_display }}{% if object.participation == object.PARTICIPATION_YES and object.published_projects.count %},
+                                    <a href='#participation-plans'>{% trans 'see participation plans' %}</a>
+                                {% endif%}
+                            </dd>
+                        </dl>
+
+                        <dl>
+                            <dt>{% trans 'Planed cost' %}</dt><dd>{{ object.cost }}€</dd>
```

Comment:
I think, cost is not required so  I guess it would be better to check if it is set, otherwise there would be None€